scripts/package-for-ci

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -eux
+
+source $(dirname $0)/version
+
+echo $TAG
+
+cd $(dirname $0)/..
+
+mkdir -p dist/artifacts
+
+REPO=rancher
+
+IMAGE=${REPO}/webhook:${TAG}
+DOCKERFILE=./package/Dockerfile
+if [ -e ${DOCKERFILE}.${ARCH} ]; then
+    DOCKERFILE=${DOCKERFILE}.${ARCH}
+fi
+
+if [[ ${USE_DOCKER_BUILDX:-0} -eq 1 ]]; then
+    docker buildx build --platform linux/amd64 -f ${DOCKERFILE} . -t ${IMAGE} 
+else
+    docker build -f ${DOCKERFILE} -t ${IMAGE} .
+fi
+echo Built ${IMAGE}
+
+docker image save rancher/webhook:$TAG -o ./dist/artifacts/webhook.img
+
+./scripts/package-helm
```

Comment:
Isn't `./scripts/package-helm` already being run in `./script/package`?

So now `make ci` is running `./scripts/package-helm` twice, correct?

Reply:
Right, the `scripts/package` should create the image and install the bits, and `package-helm` builds a chart